bin/stencil-start

```diff
@@ -228,9 +236,21 @@ function startServer() {
                 ignoreInitial: true,
                 ignored: watchIgnored.map(val => Path.join(themePath, val)),
             },
-            proxy: "localhost:" + stencilServerPort,
+            proxy: {
+                target: "localhost:" + stencilServerPort,
+                proxyReq: [
```

Comment:
I am wondering if we can do this inside the plugin itself.

Reply:
I will test

Reply:
So we could do this, but I would further argue that we shouldn't.

it looks like this:

```  
bs._options
    .get("proxy")
    .get("proxyReq")
    .push(function (proxyReq: any) {
      proxyReq.setHeader(
        "Accept-Language",
        StorefrontConfig.getInstance().getConfiguredLanguage()
      );
    });
```

the `._options` is intended to be private to browser sync.

Same issue as above in regards to initialization logic all over the place.  We could have collisions and weird overrides.  It will also be harder to track bugs.

Reply:
Since it is a plugin the logic should be self contained in the plugin itself. The only thing which should come from the client is overrides or settings params.

Reply:
With further testing this unfortunately does not seem to work if it is in the plugin.  Is this something that can be circled on in future PRs as more logic is added?